pkg/http_client.go

```
@@ -42,9 +42,16 @@ func (hcc *HttpClientConfig) BuildRoundTripper() (http.RoundTripper, error) {
 				return nil, fmt.Errorf("failed to add CA cert to pool: %v", hcc.AdditionalCACerts[i])
 			}
 		}
+		minVersion := uint16(tls.VersionTLS13)
+		switch hcc.TlsMinVersion {
+		case "1.2":
+			minVersion = uint16(tls.VersionTLS12)
+		case "1.3":
+			minVersion = uint16(tls.VersionTLS13)
+		}
```

can you have this throw an error if the user provides an invalid version?

---

Thanks Tom. Done in [4562dd3](https://github.com/semgrep/semgrep-network-broker/pull/147/commits/4562dd3db8af10476c4389a252c6e85f30eaa358)